src/geo.c

```diff
@@ -676,6 +680,9 @@ void georadiusGeneric(client *c, int srcKeyIndex, int flags) {
         return;
     }
 
+    /* Return ASAP when src key does not exist. */
+    if (zobj == NULL) goto src_key_not_exist;
```

Comment:
why use a `goto` and not just put the early return code here?

Reply:
right. I'm stupid... Done